src/daemons/rhsmcertd.c

```diff
@@ -334,6 +341,17 @@ key_file_init_config (Config * config, GKeyFile * key_file)
     else if (heal_frequency > 0) {
         config->heal_interval_seconds = heal_frequency * 60;
     }
+
+    int max_splay_minutes = get_int_from_config_file (key_file, "rhsmcertd",
+                            "maxSplayMinutes");
+    if (max_splay_minutes >= 0) {
+        if (max_splay_minutes > RAND_MAX_MINUTES) {
```

Comment:
I don't think you should change it, but I just think it's funny to think about someone setting the delay that high.